imjoy/imjoyPluginEngine.py

```diff
@@ -683,50 +697,84 @@ def connect(sid, environ):
 @sio.on("init_plugin", namespace=NAME_SPACE)
 async def on_init_plugin(sid, kwargs):
```

Comment:
This function is long... :see_no_evil:

Reply:
Yes, it's very long, I hesitate to do it right now, will have to rewrite later.